### PR TITLE
feat: add additional information to user record

### PR DESCRIPTION
**Pull request by brandonroberts**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

This PR:
- Adds additional fields to user record

Syncing information
- name (synced from GitHub)
- email  (synced from GitHub on connection, manual update after from insights)
- bio  (synced from GitHub)
- location  (synced from GitHub)
- twitter_username  (synced from GitHub)
- company  (synced from GitHub)
- display_local_time (Default `false`, manual update after from insights)
- timezone (Placeholder for future, manual update)

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: src/user/user.entity.ts [hunk 1]
```diff
@@ -115,6 +115,79 @@ export class DbUser extends BaseEntity {
...
+  })
+  @Column({
+    type: "boolean",
+    length: 30,
+    select: false,
+    default: false,
+  })
+  readonly display_local_time?: boolean;
+
+  @ApiModelProperty({
```
Comment (0-vortex) [inline src/user/user.entity.ts:180]: hard no on timezone :D

**Review by 0-vortex — COMMENTED** [review]
I don't think we can make that local timezone conversion work right now, especially since our database stores dates without timezones. While the system is usable on the frontend, we couldn't use all off the native date functionality postgres provides.

Was the above accounted for? Can we add how some of the fields will work?

Also, this PR could mention the update strategy for those fields:
- always sync from github
- sync first time from github
- edit from insights
- other combination of the above

File: src/user/user.entity.ts (hunk 1 above)
Comment (brandonroberts) [reply src/user/user.entity.ts:180]: That's why I put it as `select: false` 😉 for now. Remove it anyway?

Comment (0-vortex) [reply src/user/user.entity.ts:180]: Ok with commenting it out! 👍

**Review by 0-vortex — APPROVED** [review]
LGTM with tz commented or removed until we can settle on design 👍

**Pull request merged**